Source/Dafny/DafnyOptions.cs

```diff
@@ -1019,14 +1019,14 @@ public enum IncludesModes {
     is printed.
     If no view is specified, then everything in the module is printed.
 /showSnippets:<n>
-    0 (default) - don't show source code snippets for Dafny messages
-    1 - show a source code snippet for each Dafny message
+    0 (default) - Don't show source code snippets for Dafny messages.
```

Comment:
I would have avoided putting in periods. But if we are we should be consistent. The review does not show unchanged lines, but I see missing periods on lines 1013 and 1014 for example. Also 1003